catalog/systemd.catalog.in

```diff
@@ -119,6 +119,16 @@ Documentation: sd-login(3)
 
 A seat @SEAT_ID@ has been removed and is no longer available.
 
+-- b2bcbaf5edf948e093ce50bbea0e81ec
+Subject: The Secure Attention Key (SAK) pressed on @SEAT_ID@
+Defined-By: systemd
+Support: %SUPPORT_URL%
+Documentation: sd-login(3)
+
+The Secure Attention Key (Ctrl+Alt+Shift+Esc) was pressed on @SEAT_ID@.
+
+This emits the dbus signal SecureAttentionKey() on org.freedesktop.login1.Manager on the system bus.
```

Comment:
Nah, the message catalog is supposed to contain information for users/admins, not for developers. dbus signals are concepts for developers.

Just say something like "Pressing the SAK indicates an explicit request by the user for the system to display a secure login dialog."